lib/rules/checked-requires-onchange-or-readonly.js

```diff
@@ -41,6 +41,7 @@ function extractTargetProps(properties, keyName) {
   );
 }
 
+/** @type { import('eslint').Rule.RuleModule } */
```

Comment:
curly braces should never have spaces directly inside them
```suggestion
/** @type {import('eslint').Rule.RuleModule} */
```

Reply:
fixed in 9018532 .